storage/test/authz_models.go

```diff
@@ -15,48 +15,40 @@ import (
 
 func WriteAndReadAuthorizationModelTest(t *testing.T, datastore storage.OpenFGADatastore) {
 	ctx := context.Background()
-	store := id.Must(id.New()).String()
+	storeID := id.Must(id.New()).String()
 
-	model := &openfgapb.AuthorizationModel{
-		Id:            id.Must(id.New()).String(),
-		SchemaVersion: "1.0",
-		TypeDefinitions: []*openfgapb.TypeDefinition{
-			{
-				Type: "folder",
-				Relations: map[string]*openfgapb.Userset{
-					"viewer": {
-						Userset: &openfgapb.Userset_This{
-							This: &openfgapb.DirectUserset{},
-						},
-					},
-				},
-			},
-		},
-	}
+	t.Run("write, then read, succeeds", func(t *testing.T) {
+		model := &openfgapb.AuthorizationModel{
+			Id:              id.Must(id.New()).String(),
+			SchemaVersion:   "1.0",
```

Comment:
One thing we may consider doing is making these const in the `typesystem` package.

```
package typesystem

const (
	SchemaVersion_1_0 = "1.0"
	SchemaVersion_1_1 = "1.1"
)

```

Reply:
Good idea. Done. Thanks!